execution_engine/src/core/tracking_copy/mod.rs

```diff
@@ -400,6 +400,12 @@ impl<R: StateReader<Key, StoredValue>> TrackingCopy<R> {
                 StoredValue::ContractWasm(_) => {
                     return Ok(query.into_not_found_result(&"ContractWasm value found."));
                 }
+                StoredValue::Transfer(_) => {
+                    let _name = query.next_name();
```

Comment:
I'm not quite following the rationale here - could you explain please?

Reply:
I wasn't sure what I was doing when I first wrote this.  

Now that I better understand query and yr bringing this up, I'm wondering how we should handle cases where people give us a path when querying for a `Transfer` or `DepolyInfo`.  It wouldn't really make sense to do so.  Should would fail or just ignore the path?

Reply:
I _think_ we should fail at these two points.  If a user is querying for a `Transfer` or `DeployInfo` it should get returned above at `if query.unvisited_names.is_empty()` - i.e. it'd have to be the final component of a path I think.

Reply:
:+1:

Reply:
Okay think I have addressed this in https://github.com/CasperLabs/casper-node/pull/402/commits/a24eb67181f7ee09e821430d95e3d3a1a96f47dc.